packer/scripts/ubuntu/slurm.sh

```diff
@@ -77,14 +77,14 @@ curl -fSsL -O https://github.com/NVIDIA/enroot/releases/download/v3.4.0/enroot_3
 curl -fSsL -O https://github.com/NVIDIA/enroot/releases/download/v3.4.0/enroot+caps_3.4.0-1_${arch}.deb # optional
 apt install -y ./*.deb
 
-# Install NVIDIA container support
-distribution=$(. /etc/os-release;echo $ID$VERSION_ID) \
-         && curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | sudo gpg --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg \
-         && curl -s -L https://nvidia.github.io/libnvidia-container/$distribution/libnvidia-container.list | \
-               sed 's#deb https://#deb [signed-by=/usr/share/keyrings/nvidia-container-toolkit-keyring.gpg] https://#g' | \
-               sudo tee /etc/apt/sources.list.d/nvidia-container-toolkit.list
-apt-get update
-sudo apt install -y curl gawk jq squashfs-tools parallel
-sudo apt install -y libnvidia-container-tools pigz squashfuse # optional
+# # Install NVIDIA container support
+# distribution=$(. /etc/os-release;echo $ID$VERSION_ID) \
+#          && curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | sudo gpg --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg \
+#          && curl -s -L https://nvidia.github.io/libnvidia-container/$distribution/libnvidia-container.list | \
+#                sed 's#deb https://#deb [signed-by=/usr/share/keyrings/nvidia-container-toolkit-keyring.gpg] https://#g' | \
+#                sudo tee /etc/apt/sources.list.d/nvidia-container-toolkit.list
```

Comment:
in line 54 there is the hint that this is ubuntu20.04 only; should we make this more explicit in the directory?